phpstan.neon

```diff
@@ -2,7 +2,10 @@ parameters:
     level: 7
     checkMissingIterableValueType: false
     treatPhpDocTypesAsCertain: false
+    bootstrapFiles:
+        - config/paths.php
     paths:
         - src
+        - templates
```

Comment:
Needing to set `@var` in all the templates is going to be tedious, can we skip this change?